src/index.ts

```diff
@@ -64,10 +64,12 @@ const getBrowser = async (edgeBinaryPath: string | undefined, edgeDriverVersion:
 };
 
 const downloadDriver = async (version: string) => {
-  const versionMatcher = version === 'LATEST' ? 'LATEST_STABLE' : `LATEST_RELEASE_${version.split('.')[0]}`;
-  const response = await got.get(`${cdnUrl}/${versionMatcher}`);
-
-  version = response.body.replace(/[^\d.]/g, '');
+  const useExactEdgeDriverVersion = process.env.npm_config_use_exact_edgedriver_version || process.env.USE_EXACT_EDGE_DRIVER_VERSION;
+  if (!useExactEdgeDriverVersion) {
+    const versionMatcher = version === 'LATEST' ? 'LATEST_STABLE' : `LATEST_RELEASE_${version.split('.')[0]}`;
+    const response = await got.get(`${cdnUrl}/${versionMatcher}`);
+    version = response.body.replace(/[^\d.]/g, '');
+  }
```

Comment:
I would prefer not to add extra variable but add version matching check:

```ts
const versionMatchingRegExp = /^(\d+\.\d+\.\d+\.\d+)$/g // could be a better one, just quick look
if (version.match(versionMatchingRegExp) != version) {
  const versionMatcher = version === 'LATEST' ? 'LATEST_STABLE' : `LATEST_RELEASE_${version.split('.')[0]}`;
  const response = await got.get(`${cdnUrl}/${versionMatcher}`);
  version = response.body.replace(/[^\d.]/g, '');
}
```

And would be good to add a test in `index.spec.ts` to verify `downloadDriver` with full version

Reply:
shorter way is `(!/^(\d+\.\d+\.\d+\.\d+)$/g.test(version))`

Reply:
You expect the following specification?

- If 4-numbers full version like `85.0.564.51` is specified, the exact version is used.
- If LATEST is specified, the latest stable version is used.
- Otherwise, the latest minor version for the specified version is used.

Sounds OK to me. 

One concern is, if an existing user uses old Chrome version for 85 like `85.0.0.1`, the user's actual EdgeDriver version will regress from the latest minor version of 85 to 85.0.0.1 when the user updates the ms-edge-driver library. This can break the backward compatibility. Is this acceptable? This is no problem to me.

> And would be good to add a test

Alright.

Reply:
>If 4-numbers full version like 85.0.564.51 is specified, the exact version is used.
If LATEST is specified, the latest stable version is used.
Otherwise, the latest minor version for the specified version is used.

Exactly.
I see your request as a bug fix for originally planned and expected by user behaviour: when the exact full version is provided, it should be downloaded.

>This can break the backward compatibility. Is this acceptable?
It will be released as the next minor release, not a patch to avoid confusion.

Reply:
Hi @NozomiIto. Do you have time to push an update or you want me to take over?

Reply:
Sorry, I have been busy recently.. If you take over, it is very helpful for me.